src/explore-education-statistics-frontend/src/modules/download/components/PublicationDownloadList.tsx

```diff
@@ -53,7 +53,7 @@ function PublicationList({ publications }: Props) {
                   >
                     {`${name}${getPublicationDate(path)}`}
                   </Link>
-                  {` (${extension}, ${size})`}
+                  {` (${extension.toUpperCase()}, ${size.toUpperCase()})`}
```

Comment:
I'm not sure this is correct, the meaning of Mb and MB are different

Reply:
Happy for the UI to adjust the extension but we cant have the UI adjust the size description as 4Mb is not the same as 4MB therefore we need to handle that serverside.

Reply:
Guidance is as follows for updating the server side logic:

```
Use MB for anything over 1MB: 4MB not 4096KB.

Use KB for anything under 1MB: 569KB not 0.55MB.

Keep it as accurate as possible and up to 2 decimal places: 4.03MB.
```

Reply:
removed the adjustment from the size description

Reply:
I think if we merge this in, then assign it to dev with a comment on jira about the filesize being in bytes rather than bits?